staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -193,6 +130,106 @@ type DeltaFIFO struct {
 	emitDeltaTypeReplaced bool
 }
 
+// DeltaType is the type of a change (addition, deletion, etc)
+type DeltaType string
+
+// Change type definition
+const (
+	Added   DeltaType = "Added"
+	Updated DeltaType = "Updated"
+	Deleted DeltaType = "Deleted"
+	// Replaced is emitted when we encountered watch errors and had to do a
+	// relist. We don't know if the replaced object has changed.
+	//
+	// NOTE: Previous versions of DeltaFIFO would use Sync for Replace events
+	// as well. Hence, Replaced is only emitted when the option
+	// EmitDeltaTypeReplaced is true.
+	Replaced DeltaType = "Replaced"
+	// Sync is for synthetic events during a periodic resync.
+	Sync DeltaType = "Sync"
+)
+
+// Delta is the type stored by a DeltaFIFO. It tells you what change
```

Comment:
BTW, this text is a bit awkward too.  What a DeltaFIFO stores is `Deltas` objects; a `Delta` is a member of a `Deltas`.